userguide/network.rst

```diff
@@ -698,58 +676,47 @@ The configurable options are summarized in
    |                      |                |                                                                                                |
    +----------------------+----------------+------------------------------------------------------------------------------------------------+
    | Options              | string         | additional parameters from                                                                     |
-   |                      |                | `ifconfig(8) <http://www.freebsd.org/cgi/man.cgi?query=ifconfig>`_                             |
+   |                      |                | `ifconfig(8) <http://www.freebsd.org/cgi/man.cgi?query=ifconfig>`__                            |
    |                      |                |                                                                                                |
    +----------------------+----------------+------------------------------------------------------------------------------------------------+
 
 
-Options can be set at the lagg level using the :guilabel:`Edit`
-button, or at the individual parent interface level using the
-:guilabel:`Edit Members` button. Changes are typically made at the
-lagg level
-(:numref:`Figure %s <lagg_edit_fig>`)
-as each interface member will inherit from the lagg. To configure at
-the interface level
-(:numref:`Figure %s <lagg_member_edit_fig>`)
-instead, the configuration must be repeated for each interface within
-the lagg.
+ Options can be set at the lagg level using the :guilabel:`Edit`
+ button, or at the individual parent interface level using the
+ :guilabel:`Edit Members` button. Changes are typically made at the
+ lagg level
+ (:numref:`Figure %s <lagg_edit_fig>`)
+ as each interface member will inherit from the lagg. To configure at
+ the interface level
+ (:numref:`Figure %s <lagg_member_edit_fig>`)
+ instead, the configuration must be repeated for each interface within
+ the lagg.
 
 
-.. _LAGG_MTU:
+ .. _LAGG_MTU:
 
-Some options can only be set on the parent interfaces and are
-inherited by the lagg interface. For example, to set the MTU on a
-lagg, use :guilabel:`Edit Members` to set the MTU for each parent
-interface.
+ Some options can only be set on the parent interfaces and are
+ inherited by the lagg interface. For example, to set the MTU on a
+ lagg, use :guilabel:`Edit Members` to set the MTU for each parent
+ interface.
 
-If the MTU settings on the lagg member interfaces are not identical,
-the smallest value is used for the MTU of the entire lagg.
+ If the MTU settings on the lagg member interfaces are not identical,
+ the smallest value is used for the MTU of the entire lagg.
 
-.. note:: A reboot is required after changing the MTU to create a
+ .. note:: A reboot is required after changing the MTU to create a
    jumbo frame lagg.
 
 
-To see if the link aggregation is properly load balancing, run this
-command from :ref:`Shell`:
+ To see if the link aggregation is properly load balancing, run this
```

Comment:
"Link aggregation load balancing can be tested with:" (followed by just the code block).